tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/BuildCommand.cs

```diff
@@ -74,16 +75,35 @@ private int BuildConfigured(Dictionary<string, string> options, string repoRoot)
             var targetApi = catalog.Apis.SingleOrDefault(api => api.ProtoPath == apiPath);
             if (targetApi is null)
             {
-                Console.WriteLine($"API path '{apiPath}' is not configured for any API.");
-                return 1;
+                // Maybe what was passed in wasn't a proto path, but a library ID (as per the pipeline-state.json, so it may be a package group ID)...
+                // This is probably a temporary measure - we'll probably end up with different options for api-path vs library-id.
```

Comment:
Yep +1 for this to be temporary.

What I don't fully understand is, why we need both. There's a mapping between API and library (groups) in the other PR, so this could just take a library ID (single or group) and librarian could do the mapping from proto path to library ID with the state?

Reply:
Only if it's got state - which it won't have when doing raw generation...